src/routes/accommodationRoute.js

```diff
@@ -0,0 +1,83 @@
+import express from 'express';
+import fileUpload from 'express-fileupload';
+import path from 'path';
+import accommodationCont from '../controllers/accommodationController';
+import accMidd from '../middlewares/accommodationMiddleware';
+import userMidd from '../middlewares/userMiddlware';
+
+const app = express.Router();
+
+/**
+ * @swagger
+ * /accommodations:
+ *   post:
+ *     description: Create accommodation
+ *     produces:
+ *      - application/json
+ *     parameters:
+ *      - in: body
+ *        name: accommodation
+ *        description: Accommodation data
+ *        schema:
+ *          type: object
+ *          required:
+ *            - description
+ *            - address
+ *            - cost
+ *            - availableSpace
+ *            - services
+ *            - amenities
+ *          properties:
+ *            description:
+ *              type: string
+ *            address:
+ *              type: string
+ *            cost:
+ *              type: double
+ *            availableSpace:
+ *              type: string
+ *            services:
+ *              type: string
+ *            amenities:
+ *              type: string
+ *     responses:
+ *       201:
+ *         description: Account created successfuly
+ *         schema:
+ *           type: object
+ *           properties:
+ *             data:
+ *               $ref: '#/definitions/Accommodation'
+ *             msg:
+ *               type: string
+ *       400:
+ *         description: Wrong data sent
+ *       401:
+ *         description: You are unauthorized to access this route
+ */
+
+/**
+ * @swagger
+ * /accommodations:
+ *   get:
+ *     description: Get accommodations
+ *     produces:
+ *      - application/json
+ *     responses:
+ *       201:
+ *         description: User logged successfuly
+ *         schema:
+ *           type: array
+ *           items:
+ *             $ref: '#/definitions/Accommodation'
+ *       401:
+ *         description: You are not authorized
+ */
+const fUpload = fileUpload({
+  useTempFiles: true,
+  tempFileDir: path.join(__dirname, '../temp'),
+});
+app.post('/', fUpload, userMidd.checkToken, accMidd.checkUserSupplier, accMidd.validateAccommodation, accMidd.checkForImages, accommodationCont.addAccommodation);
```

Comment:
@lemoissonM consider destructuring or creating a new line the route exceeds the required length